docs/report.md

```diff
@@ -26,13 +26,51 @@ Our domain model consists of two data entities, which depict the attributes of a
+##### Register
+If the user has not registered before, clicking the **Register** tab redirects the user to a prompt for signing in to GitHub to continue to Chirp. After signing in with GitHub, the user is registered and logged in to Chirp, and is now an authenticated user. If the user have registered before, this information might still be stored in cookies managed by the browser, and will then not be asked to sign in with GitHub - they will simply be logged in right away and immediately turn into an authenticated user. Once authenticated, the user's GitHub username is also used as the user's Chirp username.
+Only unauthenticated users have access to the register tab.
+##### Login
+The **Login** tab does the exact same thing as the register tab - their behaviour is the same in every way.
+#### Authenticated
+![Chirp public timeline\label{Authenticated Public Timeline}](https://raw.githubusercontent.com/ITU-BDSA23-GROUP12/Chirp/main/docs/images/public%20timeline.png)
+The **authenticated** user has access to the **Public Timeline**, **My Timeline**, **Discover**, **Logout**, and **About Me** tabs.
+##### The follow mechanic
+An authenticated user has, attached to each cheep on the public timeline, a **follow** button. Uppon clicking this, the follow button is replaced by an **unfollow** button, and the user will be regarded as following the user that sent the particular cheep.
+##### My Timeline
+The **my timeline** tab is similar to the public timeline, but here only the user's own cheeps and cheeps written by users that the user is following are displayed. This tab is only accessible for authenticated users, and is only privately accessible.
+##### Discover
+Let the user of the application be A.
+The **discover** tab contains the latest cheep of each user B that is deemed interesting for A. B is deemed interesting if at least two users followed by A are following B. Here, A can browse through users that might be more relevant to A. The discover tab is only accessible to authenticated users.
+##### Logout
+The **logout** tab turns the user into an unauthenticated user. This tab is only accessible to authenticated users.
+##### About me
+The **about** me page displays information about the user. It:
+- displays the GitHub **username** and **email** used for authentication. 
+- holds a **Forget Me!** button that **deletes all information** accisiated with the user, including sent cheeps, from the application.
+- displays a list of all **cheeps written** by the user.
+- displays a list of all **users followed** by the user.
+- displays a list of all **users following** the user.
+
+The about me tab is only accessible for authenticated users, and is only privately accessible.
+##### The cheep box
+The **cheep box** is a text entry field accompanied with a **Share 🚜** button to send any text entered as a cheep. The cheep box is available on the **public timeline** page and on the **my timeline** page.
```

Comment:
Since you have been consistent with saying what is accessible for who, maybe just add that the cheep box is only active then authenticated :)